### PR TITLE
PB-4863: Add code to dump mongodb collection

### PR DESCRIPTION
**Pull request by kshithijiyer-px**
**What this PR does / why we need it**:
- [x] Add code to dump mongodb collection

Successful run without the parameter: https://aetos.pwx.purestorage.com/resultSet/testSetID/469326 
Successful run with the parameter: https://aetos.pwx.purestorage.com/resultSet/testSetID/469901 
Mainpipeline with nfs: https://aetos.pwx.purestorage.com/resultSet/testSetID/470002  

**Which issue(s) this PR fixes** (optional)
Closes #PB-4863

**Special notes for your reviewer**:
Temp code used for testing:
```
var _ = Describe("{kiyertest}", func() {

	JustBeforeEach(func() {
		StartPxBackupTorpedoTest("kiyertest", "Verify if kube-system, kube-node-lease, kube-public and Px Namespace is skipped on all namespace backup", nil, 92858, Ak, Q3FY24)
	})

	It("Verify if kube-system, kube-node-lease, kube-public and Px Namespace is skipped on all namespace backup", func() {
		Step("Check if kube-system and px namespace was backed up or not", func() {
			// Debug code to check if we see failures seen in PB-4863 and dump restoreobjects collection
			mongodbusername := os.Getenv("PX_BACKUP_MONGODB_USERNAME")
			mongodbpasswd := os.Getenv("PX_BACKUP_MONGODB_PASSWORD")
			if mongodbusername != "" && mongodbpasswd != "" {
				collectionName := "schedulepolicyobjects"
				srcClusterConfigPath, err := GetSourceClusterConfigPath()
				dash.VerifyFatal(err, nil, fmt.Sprintf("Getting kubeconfig path for source cluster %v", srcClusterConfigPath))
				err = dumpMongodbCollectionOnConsole(srcClusterConfigPath, collectionName, mongodbusername, mongodbpasswd)
				dash.VerifyFatal(err, nil, fmt.Sprintf("Dump mongodb collection Colletion name %s", collectionName))
			}
		})
	})

	JustAfterEach(func() {
	})
})
```

Dependent PR from other Project: 
https://github.com/portworx/jenkinsfiles/pull/1142

### CODE REVIEW

**Review by ak-px — APPROVED** [review]
LGTM

Are we planing to have this code in testcase just in case if we need any debugging ?

**Comment by kshithijiyer-px** [comment]
> LGTM
> 
> Are we planing to have this code in testcase just in case if we need any debugging ? 

For now it's just a need to base thing I don't want to add for every case. We can use it as and when needed.

**Review by sn-px — APPROVED** [review]
LGTM

File: tests/backup/backup_helper.go [hunk 1]
```diff
@@ -6302,3 +6302,26 @@ func isStorageClassPresent(storageClassName string) (bool, error) {
 	}
 	return false, nil
 }
+
+//dumpMongodbCollectionOnConsole Dump the collection mentioned on the console for debugging
```
Comment (mkoppal-px) [inline tests/backup/backup_helper.go:6306]: nit: there needs to be a space after `//` else GoLand will format it
nit: `Dumps`

**Review by mkoppal-px — APPROVED** [review]
LGTM except for a minor comment

File: tests/backup/backup_helper.go (hunk 1 above)
Comment (kshithijiyer-px) [reply tests/backup/backup_helper.go:6306]: Done.

**Comment by kshithijiyer-px** [comment]
> LGTM except for a minor comment

Done

**Pull request merged**